### ISSUE TITLE
web site documentation for mixtures

### ISSUE THREAD

**Issue by HightopRaven** [CoolProp/CoolProp#2427]
The CoolProp.org web site has a mixtures page that does not clearly identify the relevant stereo-isomer of R1234ze.
http://www.coolprop.org/fluid_properties/Mixtures.html

There is one instance of R1234ze(E) which is okay, but there are six instances of "R1234ZE" that do not state whether the suffix is (E) or (Z). Users must then lookup the referenced source [[158](http://www.coolprop.org/zbibliography.html#id32)] to determine which isomer of this molecule has binary interaction parameters. That publication is also vague, in most instances not using a suffix of (E) or (Z) to indicate respectively whether the trans- or cis- isomer, however the subsequent references to other publications eventually make it clear that these six instances should all be R1234ze(E).

Should the web page for mixtures be updated, despite matching the content of the cited reference source?

**Comment by ibell** [comment]
There is a long and boring story here, but I made a mistake in my paper based on a mistake in REFPROP 9 that I followed. Indeed all those that are R1234ZE should be R1234ZEE. It would be good if you could file a pull request fixing that in the source code.

**Comment by henningjp** [comment]
@ibell 
Just turn all six of them all into R1234ze(E) to match the 7th?
Looks like there's some trickery here with binary encoding of the JSON.  Just edit the dev/mixtures/mixture_binary_pairs.json file?

**Comment by henningjp** [comment]
@ibell 
Should the dev/mixtures/predifined_mixtures.JSON file be updated to replace `R1234ZE` with `R1234ZEE` for clarity?

R1234ZE has the aliases:
```
    "ALIASES": [
      "R1234ZE",
      "R1234ZEE",
      "R1234zeE",
      "R1234ZE(E)"
    ],
```

Pretty sure this is the case, but wanted to double check before I go messing around.  Is the alias R1234ze(E) preferrable as used in the one accurate instance of the BIPs that is clear?  R1234ze(E) is the name of the JSON fluid file.

**Comment by ibell** [comment]
Yes, all the R1234ZE should be replaced with R1234ZEE. Or it might make sense to replace all names with their default CoolProp names, if someone is going to go through this exercise

**Comment by henningjp** [comment]
@ibell - Curiosity came up looking at regression testing this change:

The interaction parameters with between R1234ze(E) and R32 for betaT and betaV are shown on the web site as:

![Image](https://github.com/user-attachments/assets/cd393ec1-3b68-44b4-99d8-93410c6fc8f8)

If I pull the same parameters from the code using get_fluid_param_string(), I get the same numbers:

![Image](https://github.com/user-attachments/assets/522ef500-e00e-4d85-bd87-ee9ab809a131)

Yet, the values in the JSON file are different:

![Image](https://github.com/user-attachments/assets/85f53a12-9b8f-471e-a73b-35ff5da10e34)

All the other parameters match, but not betaT or betaV.

Looking at other pairs in the JSON, some match the values on the web site and some don't. (e.g. Methane/Propane match, but Methane/CarbonDioxide don't)

Are the betaT/betaV values in the mixture_binary_pairs.JSON getting modified in some way?  Are they coming from somewhere else?

**Comment by henningjp** [comment]
Never mind.  I think I figured it out.  They are inverses, depending on which fluid is specified first.

**Comment by ibell** [comment]
Correct; 
beta_Xij = 1/beta_Xji, for X in {T,V}
gamma_Xij = gamma_Xji, for X in {T,V}
